demo/web/src/main/notebooks/Python Data Science/Functions.md

```diff
@@ -0,0 +1,68 @@
+# **Python functions in Deephaven**
+
+This notebook will demonstrate how to write a Python function that can be used in the Deephaven Query Language.
```

Comment:
sell me on functions - how will this empower me as a Deephaven user? it is hard?